src/app/login/login.component.ts

```diff
@@ -18,50 +18,60 @@ export class LoginComponent implements OnInit {
     public storage: StorageService,
     private toast: ToastController
   ) { }
+
   async ngOnInit() {
+    this.init();
+    this.setInittialFormValues();
+  }
+
+  private init() {
     this.isSubmited = false;
     this.loginForm = this.formBuilder.group({
       email: ['', [Validators.required, Validators.email]],
       password: ['', [Validators.required, Validators.minLength(5)]],
       remember: [false],
     });
-    await this.setInittialFormValues();
   }
 
   submitLogin() {
     this.isSubmited = true;
     if (this.loginForm.valid){
-      // Tell the user OK result
+
+      // Tell user OK result
       this.toast.create({
         message: 'OK',
         duration: 1800
       }).then((toastRes) => {
         toastRes.present();
       });
-      // Save credentials or delete it.
-      if (this.loginForm.controls.remember.value) {
-        this.storage.set(MAIL_KEY, this.loginForm.controls.email.value);
-        this.storage.set(PASS_KEY, this.loginForm.controls.password.value);
-      } else {
-        this.storage.remove(MAIL_KEY);
-        this.storage.remove(PASS_KEY);
-      }
+      this.saveOrRemoveCredentials();
+    }
+  }
+
+  private saveOrRemoveCredentials() {
+    if (this.loginForm.controls.remember.value) {
+      this.storage.set(SAVED_USER, {
+        email: this.loginForm.controls.email.value,
+        password: this.loginForm.controls.password.value
+      });
+    } else {
+      this.storage.remove(SAVED_USER);
+
     }
   }
 
   /**
    * Write saved values to the form.
    */
   private async setInittialFormValues() {
-    // Reading the values
-    const savedEmail = await this.storage.get(MAIL_KEY);
-    const savedPass = await this.storage.get(PASS_KEY);
+    // Reading the saved values
+    const savedValues = await this.storage.get(SAVED_USER);
 
     // Setting the values
-    this.loginForm.controls.email.setValue(savedEmail ? savedEmail : '');
-    this.loginForm.controls.password.setValue(savedPass ? savedPass : '');
+    this.loginForm.controls.email.setValue(savedValues?.email ? savedValues?.email : '');
+    this.loginForm.controls.password.setValue(savedValues?.password ? savedValues?.password : '');
```

Comment:
desde la versión 14 de nodejs podemos hacer
```suggestion
    this.loginForm.controls.password.setValue(savedValues?.password ?? '');
```